Comp2023/src/main/java/frc/robot/subsystems/Elbow.java

```diff
@@ -365,17 +357,13 @@ public void moveElbowAngleInit(ElbowAngle angle)
   {
     if (m_elbowDebug)
     {
-      m_velocity = (int) SmartDashboard.getNumber("EL_velocity", m_velocity);
-      m_acceleration = (int) SmartDashboard.getNumber("EL_acceleration", m_acceleration);
-      m_sCurveStrength = (int) SmartDashboard.getNumber("EL_sCurveStrength", m_sCurveStrength);
-
-      m_elbow.configMotionCruiseVelocity(m_velocity);
-      m_elbow.configMotionAcceleration(m_acceleration);
-      m_elbow.configMotionSCurveStrength(m_sCurveStrength);
-      m_elbow.config_kF(SLOTINDEX, ELConsts.kElbowPidKf);
-      m_elbow.config_kP(SLOTINDEX, ELConsts.kElbowPidKp);
-      m_elbow.config_kI(SLOTINDEX, ELConsts.kElbowPidKi);
-      m_elbow.config_kD(SLOTINDEX, ELConsts.kElbowPidKd);
+      m_elbow.configMotionCruiseVelocity(ELConsts.kMMVelocity);
```

Comment:
Again, I'm not sure it's even useful to put this stuff to the dashboard anymore. It is a known constant that doesn't change once we initialize the motor contollers. I'd remove these and we can add something back if needed later. It reduces smartdashboard writes.